installation/install

```diff
@@ -85,11 +85,13 @@ function handle_error
 
 function get_os_type
 {
-	which zypper &> /dev/null && os="opensuse"
 	which pacman &> /dev/null && os="archlinux"
-	which dnf &> /dev/null && os="fedora"
 	which apt-get &> /dev/null && os="debian"
+	which apt-get &> /dev/null && os="deepin"
```

Comment:
Yes Deepin is debian based but I didn't know it would have been handled automatically, my bad :)
I will try to add more useful support 👍